mantidproject/mantid
# Change sample changer group option name

**Description of work.**
This PR renames the sample changer group option to a better name because it was causing confusion for scientists. Also, the choice now is only shown when the IRIS instrument is selected.



**To test:**
1. Open Indirect Data reduction -> ISIS Energy Transfer
2. Make sure the instrument is set to IRIS
3. Load Runs 94297-94312
4. Under output options select "Sample changer grouped"
5. Click run
6. Check that outputs are grouped into 3 group workspaces each labeled as either top, mid or btm
7. Select one of the workspaces in the groups and show sample logs,
8. the SAMP_POS log should be 6.0 for btm workspaces, 85.5 for mid, and 165.0 for top



Fixes #34636. 




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
